.github/workflows/on-tag.yml

```
@@ -0,0 +1,20 @@
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{secrets.PAT_TOKEN}}
+        with:
+          tag_name: ${{github.ref}}
+          release_name: ${{github.ref}}
+          body: |
+            Automatically generated
+          draft: false
+          prerelease: false
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)